### DISCUSSION TITLE
Why email confirmation link redirects back with error code 400 and error message "Invalid email verification type"

### DISCUSSION THREAD

**Discussion by kamalsharma9392** [supabase/supabase#15396] [Questions] [answered]
When clicking on confirmation it redirects me to the home page with an error message error message "Invalid email verification type". I did check on the Auth logs there also I got the same error
![image](https://github.com/supabase/supabase/assets/38406377/9f06222f-d39b-422c-b40d-917b528a5a0e)

**Comment by GaryAustin1** [comment, marked as answer]
Hard to know with the info presented...
There is a get and post verify each has a type required in the body.
https://github.com/supabase/gotrue#get-verify
Seems to be saying that is not there or wrong type.

**Reply by kamalsharma9392** [reply]
Hi @GaryAustin1, I was able to track the issue it was an issue with my type parameter in confirm link
